.github/workflows/consumer-benchmark.yml

```
@@ -0,0 +1,53 @@
+
+permissions:
+  contents: read
+
+jobs:
+  consumer-benchmark:
+    runs-on: macos-26
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Select Xcode 26.3
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2010:warning:3:3: Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames

(shellcheck)